test/methods.jl

```diff
@@ -111,4 +119,9 @@ end
     @test axes_min_length((1:2, :)) == 1
     @test axes_min_length((1:2, 5:12)) == 1
     @test axes_min_length((10:100, 1:5)) == 2
+
+    # testing fits_indices
+    @test fits_indices("[1024:2048, 200:300]") == [200:300, 1024:2048]
+    @test fits_indices("[:, 200:300]") == [200:300, :]
+    @test fits_indices("[1024:2048, :]") == [:, 1024:2048]
```

Comment:
again, these technically aren't valid FITS indices- but keep these tests. Just add tests for `"[1:2048, 200:300]"` etc.

Reply:
Okay, but now I think we either need to change the `fits_indices` function, or the `trimview`. Currently, `trimview` only supports indices of the format `(:, some number)` or `:` at the second place. So, let's modify `trimview` to the case when `(number, number)` (one of the number still remains all rows or all columns, but it will be as range not as `:`) gets processed in it. Thoughts?

Reply:
that is a good point. I think you should modify trimview to be able to work with `1:100, 80:100` just as well as `:, 80:100`. Let me know if you have issues figuring out how to enable that.